src/ui/screens/Parachains/CrowdloanScreen.tsx

```diff
@@ -142,47 +112,39 @@ export function CrowdloanScreen() {
   );
 }
 
-function Fund({item, active, onPressContribute}: {item: Campaign; active: boolean; onPressContribute: () => void}) {
-  const formatBalance = useFormatBalance();
-  const {cap, raised} = item.info;
-  const endpoints = useParaEndpoints(item.paraId);
+function Fund({item, active, onPressContribute}: {item: Crowdloan; active: boolean; onPressContribute: () => void}) {
   const navigation = useNavigation<NavigationProp<CrowdloansStackParamList>>();
   const {colors} = useTheme();
+  const {currentNetwork} = useContext(NetworkContext);
 
-  const lastEndpoint = endpoints?.[endpoints.length - 1] as LinkOption;
-  const text = lastEndpoint?.text ?? `#${item.paraId.toString()}`;
-
-  const percentage = cap.isZero() ? 100 : raised.muln(10000).div(cap).toNumber() / 10000;
+  const isLitentryParachain = currentNetwork.key === 'polkadot' && item.paraId === '2013';
+  const isLitmusParachain = currentNetwork.key === 'kusama' && item.paraId === '2106';
+  const isSpecial = isLitentryParachain || isLitmusParachain;
```

Comment:
This logic is already implemented in the backend. Can we add extra status flags from the graph?